spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/hive/SparkHiveTable.java

```diff
@@ -8,23 +8,86 @@
 import com.datastrato.gravitino.rel.Table;
 import com.datastrato.gravitino.spark.connector.PropertiesConverter;
 import com.datastrato.gravitino.spark.connector.SparkTransformConverter;
-import com.datastrato.gravitino.spark.connector.table.SparkBaseTable;
+import com.datastrato.gravitino.spark.connector.utils.GravitinoTableInfoHelper;
+import com.google.common.annotations.VisibleForTesting;
+import java.util.Map;
+import java.util.Set;
+import org.apache.kyuubi.spark.connector.hive.HiveTable;
+import org.apache.kyuubi.spark.connector.hive.HiveTableCatalog;
+import org.apache.spark.sql.SparkSession;
 import org.apache.spark.sql.connector.catalog.Identifier;
+import org.apache.spark.sql.connector.catalog.SupportsRead;
+import org.apache.spark.sql.connector.catalog.SupportsWrite;
+import org.apache.spark.sql.connector.catalog.TableCapability;
 import org.apache.spark.sql.connector.catalog.TableCatalog;
+import org.apache.spark.sql.connector.expressions.Transform;
+import org.apache.spark.sql.connector.read.ScanBuilder;
+import org.apache.spark.sql.connector.write.LogicalWriteInfo;
+import org.apache.spark.sql.connector.write.WriteBuilder;
+import org.apache.spark.sql.types.StructType;
+import org.apache.spark.sql.util.CaseInsensitiveStringMap;
+
+/** Keep consistent behavior with the SparkIcebergTable */
+public class SparkHiveTable extends HiveTable {
+
+  private GravitinoTableInfoHelper gravitinoTableInfoHelper;
+  private org.apache.spark.sql.connector.catalog.Table sparkTable;
```

Comment:
but it seems inconsistent with SparkIcebergTable's behavior. cc @FANNG1

Reply:
why?

Reply:
because `SparkIcebergTable` need override `newScanBuilder` to support `time travel`.

Reply:
`SparkIcebergTable` extends `SparkTable` already, so it could use the default implement of `newScanBuilder` to support `time travel`, and doesn't need to override `newScanBuilder`

Reply:
ok

Reply:
updated.